src/collectors/portcheck/portcheck.py

```diff
@@ -0,0 +1,108 @@
+"""
+The PortCheckCollector checks ports listed in config file.
+
+##### Dependencies
+
+* psutil
+
+Example config file PortCheckCollector.conf
+
+```
+enabled = True
+ttl = 120
+
+[port]
+[[echo]]
```

Comment:
Is there any restriction on port name?

Reply:
what restrictions are you thinking of? I didn't put any restriction now